core/src/com/unciv/logic/battle/CityCombatant.kt

```diff
@@ -26,15 +27,15 @@ class CityCombatant(val city: CityInfo) : ICombatant {
     }
 
     override fun getUnitType(): UnitType = UnitType.City
-    override fun getAttackingStrength(): Int = getCityStrength()*2/5 // I remember reading this but I don't recall where
```

Comment:
I found the post with the original code here:
https://forums.civfanatics.com/threads/formula-for-city-combat-strength.535944/

So yeah, it's 0.4f

Reply:
you check this web, it said that we can find the word "CITY_RANGED_ATTACK_STRENGTH_MULTIPLIER" in original game file "C:\Program Files\Steam\steamapps\common\sid meier's civilization v\assets\Gameplay\XML\GlobalDefines.xml", it's 50, in other words, 0.5f.
http://www.civclub.net/bbs/forum.php?mod=viewthread&tid=87771

Reply:
sorry, we should find the word "CITY_RANGED_ATTACK_STRENGTH_MULTIPLIER" in "\Assets\DLC\Expansion\Gameplay\XML\GlobalDefines.xml", above file it's previous.
now unciv is based in dlc game. in the dlc, it's 0.75.

Reply:
Aha, so this is a Vanilla vs G&K issue? Interesting, thanks!